### PR TITLE
Added a few handlers

### PR DESCRIPTION
**Pull request by gldeng**

### CODE REVIEW

File: AElf.OS.Network.Grpc/GrpcUrl.cs [hunk 1]
```diff
@@ -17,6 +17,11 @@ public static GrpcUrl Parse(string address)
             return url;
         }
         
+        public string ToIpPortFormat()
```
Comment (loning) [inline AElf.OS.Network.Grpc/GrpcUrl.cs]: You can use System.Uri or System.Net.IpAddress

no need to implement by your self

Comment (loning) [reply AElf.OS.Network.Grpc/GrpcUrl.cs]: @samfrb

File: AElf.OS.Network.Grpc/GrpcPeerPool.cs [hunk 2]
```diff
@@ -134,7 +134,7 @@ public GrpcPeer FindPeer(string peerEndpoint, byte[] publicKey = null)
             IEnumerable<GrpcPeer> toFind = _authenticatedPeers;
 
             if (!string.IsNullOrWhiteSpace(peerEndpoint))
-                toFind = toFind.Where(p => p.PeerAddress == peerEndpoint);
+                toFind = toFind.Where(p => p.PeerAddress == peerEndpoint || peerEndpoint.EndsWith(p.RemoteListenPort));
```
Comment (loning) [inline AElf.OS.Network.Grpc/GrpcPeerPool.cs]: do not use port to match peer.

just public key

File: AElf.OS.Network.Grpc/GrpcUrl.cs (hunk 1 above)
Comment (samfrb) [reply AElf.OS.Network.Grpc/GrpcUrl.cs]: will be solved by #940

File: AElf.OS.Network.Grpc/GrpcPeerPool.cs (hunk 2 above)
Comment (samfrb) [reply AElf.OS.Network.Grpc/GrpcPeerPool.cs]: will be solved by #940

**Pull request merged**